### PR TITLE
Chore cleanup

### PR DESCRIPTION
**Pull request by harrisonhjones**
Some small cleanup:

1. Breaks the README up into two files:
    1. README.md - our actual readme.
    1. README-CRA.md - the original CRA readme.
1. Renames the app from `yata-web` to `yata-web-app`.
1. Adds a `.prettierrc` to keep formatting consistent across multiple devs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,99 +28,3 @@ TODO: we should be able to specify this file as an input running `npm start`.
...
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
-
-### `npm test`
```
Comment (TheYeung1) [inline README.md:53]: can we leave in something about testing?

File: README.md [hunk 2]
```diff
@@ -28,99 +28,3 @@ TODO: we should be able to specify this file as an input running `npm start`.
...
-# Getting Started with Create React App
-
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm run format`
```
Comment (TheYeung1) [inline README.md:41]: can we leave in something about formatting?

**Review by TheYeung1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (harrisonhjones) [reply README.md:53]: Yup. Ok if I leave it in the CRA readme as well?

File: README.md (hunk 2 above)
Comment (harrisonhjones) [reply README.md:41]: Yup. Ok if I leave it in the CRA readme as well?

File: README.md (hunk 1 above)
Comment (TheYeung1) [reply README.md]: yup

File: README.md (hunk 2 above)
Comment (TheYeung1) [reply README.md]: yup

**Pull request merged**